sql/_05_plcsql/_01_testspec/_02_declaration/_02_variable/_04_deeping/_02_char/cases/01_char_basic.sql

```diff
@@ -0,0 +1,56 @@
+--+ server-message on
+-- Verified for CBRD-24901
+
+create or replace procedure char_test1(t_query varchar) as
+	ch_1 character(20);
+begin
+	execute immediate t_query into ch_1;
+	
+	dbms_output.put_line('append spaces until filling char size');
+	dbms_output.put_line('char: "' || ch_1 ||'"');
+end;
+
+call char_test1('select cast(''abc'' as char(3)) from dual;');
+-- error, size over
+call char_test1('select cast(''abc'' as char(21)) from dual;');
+
+
+create or replace procedure char_test2() as
+	num_1 character(20) := 3;
+	num_2 character(5) := 8;
```

Comment:
num_1 character(20) := 2;
	num_2 character(5) := 10;

Reply:
num_1, num_2 is char type.
this case is not auto-cast by integer.
Therefore I add a num_1 > num_2 and num_2 > num_1 on this test case.

Reply:
add_plcsql_04